src/v3/src/components/PasswordRequirements/Icon.tsx

```diff
@@ -38,7 +38,7 @@ const Icon: FunctionComponent<PasswordRequirementIconProps> = (
       display="flex"
     >
       <OdyIcon
-        // TODO: OKTA-556721 - Create and use loc string here for requirement status
+        aria-hidden
         titleAccess={status}
```

Comment:
kept the title here since it is utilized in the integration tests to determine if a password is complete or not.